scripts/ci/run-jenkins.sh

```diff
@@ -325,16 +329,24 @@ if [[ ${CI_TAGS} == *'webassembly'* ]] || [[ ${CI_TAGS} == *'wasm'* ]];
         echo "ENABLE_WASM_DYNAMIC_RUNTIME=1" >> sdks/Make.config
         #echo "ENABLE_WASM_THREADS=1" >> sdks/Make.config
 
-	   export aot_test_suites="System.Core"
-	   export mixed_test_suites="System.Core"
-	   export xunit_test_suites="System.Core corlib"
-
-	   ${TESTCMD} --label=provision --timeout=20m --fatal $gnumake --output-sync=recurse --trace -C sdks/builds provision-wasm
-
-	   ${TESTCMD} --label=configure --timeout=180m --fatal $gnumake -j ${CI_CPU_COUNT} --output-sync=recurse --trace -C sdks/builds configure-wasm NINJA=
-	   ${TESTCMD} --label=build     --timeout=180m --fatal $gnumake -j ${CI_CPU_COUNT} --output-sync=recurse --trace -C sdks/builds build-wasm     NINJA=
-	   ${TESTCMD} --label=archive   --timeout=180m --fatal $gnumake -j ${CI_CPU_COUNT} --output-sync=recurse --trace -C sdks/builds archive-wasm   NINJA=
-
+        export aot_test_suites="System.Core"
+        export mixed_test_suites="System.Core"
+        export xunit_test_suites="System.Core corlib"
+
+        ${TESTCMD} --label=provision --timeout=20m --fatal $gnumake --output-sync=recurse --trace -C sdks/builds provision-wasm
+
+        if [[ ${CI_TAGS} == *'osx-amd64'* ]]; then
+            ${TESTCMD} --label=build-runtime     --timeout=180m --fatal $gnumake -j ${CI_CPU_COUNT} --output-sync=recurse --trace -C sdks/wasm runtime
```

Comment:
Why doesn't the other version work on osx ?

Reply:
@MaximLipnin we should check whether this is still necessary with the latest PR job config.